source/examples/fibcmd/CMakeLists.txt

```diff
@@ -59,6 +59,9 @@ target_include_directories(${target}
     PRIVATE
     ${DEFAULT_INCLUDE_DIRECTORIES}
     ${PROJECT_BINARY_DIR}/source/include
+    
+    $<TARGET_PROPERTY:${META_PROJECT_NAME}::baselib,INTERFACE_INCLUDE_DIRECTORIES>
```

Comment:
@sbusch42 These lines seem superfluous with the dependent target system cmake establishes, but somehow the API of cmake doesn't allow to retrieve additional include directories of dependent targets. The health check system is already useful without those extra headers but without them we probably have to reconfigure the scripts to silence missing includes.
What is your opinion here?